2006-06-01	Arnaud Charlet	<charlet@adacore.com>

	* configure: work around libtool bug on ia64 hpux
	See http://bugzilla.gnome.org/show_bug.cgi?id=343485

2006-02-23	Arnaud Charlet	<charlet@adacore.com>

	* gdk/x11/gdkasync.c: #define ANSICPP on Solaris, to work around
	bug in X11/Xmd.h on Solaris 8.

2006-01-04	Arnaud Charlet	<charlet@adacore.com>

	* gtk/gtktextview.c (gtk_text_view_start_selection_dnd): Disabled,
	since not functional.
	For C928-003.

2003-11-27	Arnaud Charlet	<charlet@adacore.com>

	* gtk/gtktextview.c (incremental_validate_callback): Increase amount
	of characters validated at each call, this gives a better behavior
	when loading big files.
	Submitted at http://bugzilla.gnome.org/show_bug.cgi?id=332057

--- gtk/gtktextview.c.old	2006-02-17 16:11:04.000000000 +0100
+++ gtk/gtktextview.c	2006-02-20 17:08:54.691581240 +0100
@@ -3273,7 +3273,7 @@ incremental_validate_callback (gpointer 
   
   DV(g_print(G_STRLOC"\n"));
   
-  gtk_text_layout_validate (text_view->layout, 2000);
+  gtk_text_layout_validate (text_view->layout, 20000);
 
   gtk_text_view_update_adjustments (text_view);
   
@@ -6028,6 +6028,8 @@ gtk_text_view_start_selection_dnd (GtkTe
   GdkDragContext *context;
   GtkTargetList  *target_list;
 
+  return;
+
   text_view->drag_start_x = -1;
   text_view->drag_start_y = -1;
   text_view->pending_place_cursor_button = 0;
--- gdk/x11/gdkasync.c.old	Thu Feb 23 11:30:46 2006
+++ gdk/x11/gdkasync.c	Thu Feb 23 11:48:14 2006
@@ -47,6 +47,11 @@ in this Software without prior written a
 #ifdef NEED_XIPROTO_H_FOR_XREPLY
 #include <X11/extensions/XIproto.h>
 #endif
+
+#if defined (sun) && defined (__SVR4)
+#define ANSICPP
+#endif
+
 #include <X11/Xlibint.h>
 #include "gdkasync.h"
 #include "gdkx.h"
*** configure.old	Thu Jun  1 04:35:04 2006
--- configure	Thu Jun  1 04:35:26 2006
***************
*** 7819,7825 ****
  
  	case $host_cpu in
  	hppa*64*|ia64*)
! 	  hardcode_libdir_flag_spec_ld='+b $libdir'
  	  hardcode_direct=no
  	  hardcode_shlibpath_var=no
  	  ;;
--- 7819,7825 ----
  
  	case $host_cpu in
  	hppa*64*|ia64*)
! 	  hardcode_libdir_flag_spec_ld='${wl}+b ${wl}$libdir'
  	  hardcode_direct=no
  	  hardcode_shlibpath_var=no
  	  ;;
